app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-email-signup {
+  @include govuk-responsive-margin(8, "top");
```

Comment:
We try to aim for a margin-bottom approach to everything, particularly components. Would it be possible to remove this margin and add it to whatever appears above the component? Obviously this'll be more work (if possible), sorry.